css/stylesheet.css

```diff
@@ -156,3 +161,43 @@ h4 {
     border: 2px solid #F8D5E2;
 }
 
+.contacts-hometop {
+    background-color: #cee4c7;
+}
+
+.contacts-social a {
+    color: #cee4c7;
+}
+
+.contacts-social a:hover {
+    color: #F8D5E2;
+}
+
+.contacts-hometop nav a:hover {
+    color: #F8D5E2;
+}
+
+.contacts {
+    width: 30%;
```

Comment:
this % width is cutting off your email address at certain screen sizes (see comment below for fix)